src/app/components/header/header-menu.model.ts

```diff
@@ -0,0 +1,288 @@
+          text: 'OIT',
+          href: 'javascript:'
+        },
+      ]
+    },
+    {
+      container: 'col-md-4 col-sm-4',
+      type: 'paragraph',
+      header: {
+        text: 'Latest News',
+        icon: 'fa fa-newspaper-o',
+      },
+      items: [
+        {
+          /* tslint:disable:max-line-length */
+          text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis libero purus, fermentum at libero convallis, auctor dignissim mauris. Nunc laoreet pellentesque turpis sodales ornare. Nunc vestibulum nunc lorem, at sodales velit malesuada congue. Nam est tortor, tincidunt sit amet eros vitae, aliquam finibus mauris.',
+          href: null
+        },
+        {
+          text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis libero purus, fermentum at libero convallis, auctor dignissim mauris. Nunc laoreet pellentesque turpis sodales ornare. Nunc vestibulum nunc lorem, at sodales velit malesuada congue. Nam est tortor, tincidunt sit amet eros vitae, aliquam finibus mauris.',
+          href: null
+          /* tslint:enable:max-line-length */
+        }
+      ]
+    },
+  ]
+};
+
+export const NotificationMenu = {
+  container: 'bell',
+  menu: {
+    id: 'media-list',
+    name: 'header',
+    icon: 'fa fa-bell-o f-s-14',
+  },
+  // Todo: determine data structure of /notifications API endpoint
+  items: [
+    {
+      icon: {
+        background: 'red'
+      },
+      link: 'javascript:',
+      text: 'Violations Reports Updated',
+    },
+    {
+      icon: {
+        thumbnail: 'assets/img/user-ka.jpg'
+      },
+      link: 'javascript:',
+      text: 'Violations Reports Updated'
+    },
+    {
+      icon: {
+        thumbnail: 'assets/img/user-mj.jpg'
+      },
+      link: 'javascript:',
+      text: 'Violations Reports Updated'
+    },
+    {
+      icon: {
+        background: 'green'
+      },
+      link: 'javascript:',
+      text: 'Violations Reports Updated'
+    }
+  ]
+};
+
+export const LanguageMenu = {
+  container: 'navbar-language',
+  menu: {
+    id: 'language',
+    name: 'header',
+  },
+  selected: {
+    icon: 'flag-icon flag-icon-us',
+    title: 'us',
+    text: 'EN'
+  },
+  // Todo: determine data structure of /settings/language API endpoint
+  items: [
+    {
+      icon: 'flag-icon flag-icon-us',
+      title: 'us',
+      text: 'English'
+    },
+    {
+      icon: 'flag-icon flag-icon-es',
+      title: 'es',
+      text: 'Spanish'
+    },
+    {
+      icon: 'flag-icon flag-icon-fr',
+      title: 'fr',
+      text: 'French'
+    },
+    {
+      icon: 'flag-icon flag-icon-ru',
+      title: 'ru',
+      text: 'Russian'
+    },
+    {
+      icon: 'flag-icon flag-icon-in',
+      title: 'in',
+      text: 'Hindi'
+    }
+  ]
+}
+
+
+export const UserMenu = {
+  container: 'navbar-user',
+  menu: {
+    id: 'user',
+    name: 'header',
+  },
+  items: [
+    {
+      icon: 'fa fa-pencil-square-o',
+      text: 'Edit Profile',
+      link: '#/app/extra/profile'
```

Comment:
I was just placing the same value from what header.component.html had, but I did removed hash from the url string